muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/base/mixins.scss

```diff
@@ -102,6 +102,22 @@
   }
 }
 
+@mixin form-element-input-clear-search-decoration {
+  align-items: center;
+  color: $color-search-clear-icon;
+  display: flex;
+  font-size: 0.6rem;
+  height: 1.8rem;
+  justify-content: center;
+  width: 1.8rem;
+
+  @include breakpoint($breakpoint-pad) {
+    font-size: 0.6rem;
```

Comment:
`font-size`, `height` and `width` declarations does not seem necessary here as they don't change at all from the "root" values.